### DISCUSSION TITLE
new version 2.12

### DISCUSSION THREAD

**Discussion by nixrand** [cagnulein/qdomyos-zwift#1319] [General]
Hi,

When will the new version will be released?  I  have a Treadmill BH boxster (currently only supported on beta version  2.12 with git) but not in android/iphone and so I can't use it on mobile...

**Comment by cagnulein** [comment]
Hi @nixrand the 2.12 on android is already available. Don't you get it?

**Reply by nixrand** [reply]
I've only checked iphone version, I will take a look on android Thanks!

**Comment by cagnulein** [comment]
yes for ios i'm waiting a confirmation from an user

Il giorno lun 27 feb 2023 alle 05:28 nixrand ***@***.***> ha
scritto:

> I've only checked iphone version, I will take a look on android Thanks!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/cagnulein/qdomyos-zwift/discussions/1319#discussioncomment-5118070>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAALYWFPI7NESENIJUSWMADWZQUN7ANCNFSM6AAAAAAVIWDVFY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> com>
>
-- 
Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud
